doc/README.md

```diff
@@ -97,22 +97,36 @@ cd [COURSE-TEMPLATES PROJECT ROOT]
 
 ## Making changes to Sass or JavaScript files
 
-1. Ensure you followed the steps above and have the a-plus code mounted within the `docker-compose.yml` file and the containers running with `./docker-up.sh`.
-2. In order to see changes in styles when working on the SCSS files, open up another terminal and run the following in the a-plus project root:
+Make sure you update the bootstrap3 package, and make it available under the **vendor** directory
 
+```sh
+$ cd [A-PLUS PROJECT ROOT]
+$ ./dev_assets_run_npm.sh bootstrap3 update
```

Comment:
I thought that the Bootsrap3 package would be installed by the `drone exec` command, which you should execute in the beginning anyway. Is this new addition needed for updating the Bootstrap version after it has been downloaded?

Reply:
OMG!. You are right, it seems that I was using another drone package 😳. Anyway, I think, it will be good to link the drone package that needs to be installed. I will make the corresponding changes.

Reply:
What does the command `./dev_assets_run_npm.sh bootstrap3 update` do? Update Bootstrap3 patches if there is a newer version? Since Boostrap3 has reached end of life, there won't be updates for it.

Reply:
I think that this script only executes the `npm install` and create a temporary  **node_modules** folder which are used to retrieve the boostrap3 package and then that package is added to the **vendors** folder.

Reply:
`npm install` is executed by `drone exec` and it also copies the Bootstrap3 files to the `assets` folder with the `assets_src/bootstrap3/install.sh` script, while `./dev_assets_run_npm.sh bootstrap3 update` updates the package version with `npm update`, but Bootstrap3 won't have new releases anymore.